### PR TITLE
Fix rendering variables

### PR DESCRIPTION
**Pull request by afarid**
**Issue:**
When adding Prometheus alert  labels like *{{ $labels.team }}*, it will be rendered as {{ ${labels}.team }}

The fix will keep the alert labels the same {{ $labels.team }} after templated is rendered

### CODE REVIEW

**Review by msharbaji — APPROVED** [review]

**Comment by msharbaji** [comment]
lgtm

**Review by msharbaji — APPROVED** [review]

**Comment by msharbaji** [comment]
@afarid you might need to check the conflict with default branch, and then can approved and merged :)

File: .estafette.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ stages:
     image: extensions/docker:dev
     action: build
     inline: |
-      FROM gcr.io/google.com/cloudsdktool/cloud-sdk:405.0.0-alpine
+      FROM gcr.io/google.com/cloudsdktool/cloud-sdk:410.0.0-alpine
```
Comment (ivanovaleksandar) [inline .estafette.yaml:29]: why dont use `418` instead? https://console.cloud.google.com/gcr/images/google.com:cloudsdktool/GLOBAL/cloud-sdk

**Review by ivanovaleksandar — CHANGES_REQUESTED** [review]

File: .estafette.yaml (hunk 1 above)
Comment (afarid) [reply .estafette.yaml:29]: There is breaking changes with latest image,  gke-gcloud-auth-plugin is longer exist,  I will create another task for this

Comment (ivanovaleksandar) [reply .estafette.yaml:29]: fixed with the installation of the gcloud auth plugin

**Review by ivanovaleksandar — APPROVED** [review]

**Pull request merged**